src/Data/Argonaut/Extension/Date.purs

```diff
@@ -0,0 +1,16 @@
+module Data.Argonaut.Extension.Date where
+
+import Data.Date (JSDate(), Date(), toJSDate)
+import Data.Argonaut.Core
+import Data.Argonaut.Encode
+
+
+foreign import jsDateToJson """
+function jsDateToJson(a) {return a;}
+""" :: JSDate -> Json
+
+
+instance dateEncodeJson :: EncodeJson Date where
```

Comment:
This is an orphan instance. Please submit a Pull Request to purescript-argonaut with EncodeJson / DecodeJson instances for purescript-datetime's type.